tests/jballerina-unit-test/src/test/resources/test-src/statements/onfail/on-fail-clause.bal

```diff
@@ -74,6 +78,66 @@ function testMultiLevelOnFail() {
     assertEquality(" -> Before error thrown,  -> Error caught at level #1 -> Before error thrown,  -> Error caught at level #1", str);
 }
 
+function testMultiLevelOnFailWithoutVariableVariationOne() {
+    int i = 0;
+    string str = "";
+
+    while (i < 2) {
+        do {
+            str += " -> Before error thrown, ";
+            fail getError();
+        } on fail {
+            str += " -> Error caught at level #1";
+        }
+
+        i = i + 1;
+    } on fail error e {
+        str += " -> Error caught at levet #2";
+    }
+
+    assertEquality(" -> Before error thrown,  -> Error caught at level #1 -> Before error thrown,  -> Error caught at level #1", str);
```

Comment:
These lengthy concatenations are hard to follow IMO. Just a letter or digit would be better IMO.